ldapauth.go

```diff
@@ -544,3 +566,19 @@ func LogConfigParams(config *Config) {
 		LoggerDEBUG.Printf(fmt.Sprint(typeOfS.Field(i).Name, " => '", v.Field(i).Interface(), "'"))
 	}
 }
+
+// retrieve a secret value from environment variable or secret on the FS
+func getSecret(label string) string {
+	bindPassword := os.Getenv(strings.ToUpper(label))
```

Comment:
getSecret is a generic method to retrieve a Docker secret, so we could probably find a better name for the variable `bindPassword`